http/analysis-service/templates/cluster/schedule.html

```diff
@@ -0,0 +1,168 @@
+{% import 'macros.html' as macros with context %}
+{% extends "base.html" %}
+
+{% block cluster_schedule_active %} class="active" {% endblock %}
+
+{% block title %}
+<h1>Schedule a Spark job</h1>
+<p>
+  <img width="48" src="{{ url_for('static', filename='schedule.png') }}" />
+  Run a Spark analysis on a scheduled basis.
+</p>
+<p>
+  The output of the analysis and the evaluated IPython notebook will be published in Amazon S3.
+  For data that will be used to power a dashboard, be sure to make the output public below.
+</p>
+{% endblock %}
+
+{% block content %}
+{% if jobs %}
+<div class="jobs">
+  <p class="lead">
+    Your currently scheduled jobs:
+  </p>
+  {% for job in jobs %}
+  <div class="row">
+    <div class="col-md-2"><p>Job {{ job.id }}: {{ job.name }}</p></div>
+    <div class="col-md-10">
+      <a href="{{ url_for('cluster_edit_scheduled_job', job_id=job.id) }}">Edit</a>
+      |
+      <a href="{{ url_for('cluster_delete_scheduled_job', job_id=job.id) }}">Delete</a>
+      |
+      <a href="{{ url_for('cluster_view_job_logs', job_id=job.id) }}">View Logs</a>
+      |
+      <a href="{{ url_for('cluster_view_job_data', job_id=job.id) }}">View Data</a>
+    </div>
+  </div>
+  {% endfor %}
+</div>
+<hr/>
+{% endif %}
+
+{% set form_target = url_for('cluster_create_scheduled_job') %}
+{% set is_update = False %}
+<div class="form-label">
+  {% if values and values["job-id"] %}
+  {% set is_update = True %}
+  {% set form_target = url_for('cluster_edit_scheduled_job', job_id=values["job-id"]) %}
+  <p class="lead">
+    Edit job <strong>{{ values["job-id"] }}</strong>:
+    <strong>{{ values["job-name"] }}</strong>
+  </p>
+  {% else %}
+  <p class="lead">Schedule a new job:</p>
+  {% endif %}
+</div>
+<form action="{{ form_target }}" method="POST" enctype="multipart/form-data">
+  {% if is_update %}
+  <input type="hidden" name="job-id" value="{{ values['job-id'] }}" />
+  <input type="hidden" name="job-name" value="{{ values['job-name'] }}" />
+  {% endif %}
+  <table>
+    {% if is_update %}
+
+    {% call macros.make_input('code-uri', 'S3 Code URI', required=False) %}
+    The location of the IPython notebook in S3. Either update this URI or upload a new
+    notebook below, but not both at the same time.
+    {% endcall %}
+
+    {% call macros.make_input('code', 'Code Tarball', type='file', required=False) %}
```

Comment:
`s/Code Tarball/IPython Notebook (.ipynb)/`, and update the description + example value.